modules/nf-core/samtools/merge/main.nf

```diff
@@ -1,6 +1,6 @@
 process SAMTOOLS_MERGE {
     tag "$meta.id"
-    label 'process_low'
+    label 'process_medium'
```

Comment:
alternatively you could modify the base.config file to something like this
```
withName: '.*:SAMTOOLS_MERGE' {
    label = 'process_medium'
}
```
This should update the resources used in all the samtools merge processes regardless of whether you use an alias or not. But it's best to confirm of course.
https://www.nextflow.io/docs/latest/config.html#selector-priority

or if you want to do it for a specific subworkflow you can add it to that config

Reply:
I like your suggestion better, but would this be overwritten by another config with `-c`, let's say: 
```
withName: '.*:SAMTOOLS_MERGE' {
    time = xxx
}
```
or would `label = 'process_medium'` then be kept?

Reply:
Setting time should set tima and keep the rest of the resources that are set with process_medium label. So the rest of the settings that comes with process medium like cpu should be kept.

Reply:
Alright, sounds good. Maybe I should update it on nf-core instead, I would think most systems could make use of more than 2 threads when merging BAM-files.